src/sentry/notifications/utils/__init__.py

```diff
@@ -146,7 +171,7 @@ def get_group_settings_link(
         query_params["environment"] = environment
     if alert_type:
         query_params["alert_type"] = alert_type
-    if alert_timestamp:
+    if alert_timestamp_str:
```

Comment:
This `if` is probably unnecessary, since we'll always have a value for `alert_timestamp_str`